server/helpers/sms.js

```diff
@@ -0,0 +1,16 @@
+import axios from 'axios';
+
+const { KUDI_SMS_USERNAME, KUDI_SMS_PASSWORD, KUDI_SMS_SENDERNAME } = process.env;
```

Comment:
I think we should pursue using the same sendername everywhere. If it is different in the future, we can manually add it.
```suggestion
const { KUDI_SMS_USERNAME, KUDI_SMS_PASSWORD, SMS_SENDERNAME } = process.env;
```